features/step_definitions/cli.rb

```diff
@@ -23,6 +23,23 @@
   end
 end
 
+When /^I run the :([a-z_]*?)( background)? client command with importmode if supported with:$/ do |yaml_key, background, table|
+  opts = table.raw
+  if env.version_ge("4.14", user: user)
+    opts = table.raw << [ "import_mode" , "PreserveOriginal" ]
+  end
+  if background
+    @result = user.cli_exec(
+      yaml_key.to_sym,
+      opts_array_process(opts) << [ :_background, true ]
+    )
+    @bg_rulesresults << @result
+    @bg_processes << @result[:process_object]
+  else
+    @result = user.cli_exec(yaml_key.to_sym, opts_array_process(opts))
+  end
```

Comment:
No need to duplicate the `if background...end` block. We can use `step "blabla"` to reuse existing implementations.

Reply:
@liangxia thanks a lot, fixed, PTAL!, running pass in my local